Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -236,28 +266,54 @@ - (void)configureWithThread:(ThreadViewModel *)thread
                                      [self.unreadBadge autoSetDimension:ALDimensionHeight toSize:unreadBadgeHeight],
                                  ]];
                              }];
-
-        const CGFloat kMinVMargin = 5;
-        [self.viewConstraints addObjectsFromArray:@[
-            // Horizontally, badge is inserted after the tail of the payloadView, pushing back the date *and* snippet
-            // view
-            [self.payloadView autoPinEdge:ALEdgeTrailing
-                                   toEdge:ALEdgeLeading
-                                   ofView:self.unreadBadge
-                               withOffset:-self.topRowHSpacing],
-            [self.unreadBadge autoPinTrailingToSuperviewMargin],
-            [self.unreadBadge autoPinEdgeToSuperviewEdge:ALEdgeTop
-                                               withInset:kMinVMargin
-                                                relation:NSLayoutRelationGreaterThanOrEqual],
-            [self.unreadBadge autoPinEdgeToSuperviewEdge:ALEdgeBottom
-                                               withInset:kMinVMargin
-                                                relation:NSLayoutRelationGreaterThanOrEqual],
-
-            // Vertically, badge is positioned vertically by aligning it's label *subview's* baseline.
-            // This allows us a single visual baseline of text across the top row across [name, dateTime,
-            // optional(unread count)]
-            [self.unreadLabel autoAlignAxis:ALAxisBaseline toSameAxisOfView:self.dateTimeLabel]
-        ]];
+    } else {
+        UIImage *_Nullable statusIndicatorImage = nil;
+        UIColor *messageStatusViewTintColor = textColor;
+        BOOL shouldAnimateStatusIcon = NO;
+        if (overrideSnippet == nil && [self.thread.lastMessageForInbox isKindOfClass:[TSOutgoingMessage class]]) {
```

Comment:
overrideSnippet is always nil at this point based on line 233, should this be in assert?

Reply:
Good point, removed.